src/main/resources/config.yml

```diff
@@ -576,6 +576,12 @@ integration:
     #Should we delete the shop in the land when the shop owner lose the permission
     #in this land?
     delete-on-lose-permission: false
+  fabledskyblock:
```

Comment:
Please also add config update code in org.maxgamer.quickshop.QuickShop#updateConfig